db/schema.rb

```diff
@@ -950,12 +936,8 @@
   add_foreign_key "posts", "users", column: "target_user_id"
   add_foreign_key "profile_links", "profile_link_sites"
   add_foreign_key "profile_links", "users"
-  add_foreign_key "quote_likes", "quotes"
-  add_foreign_key "quote_likes", "users"
-  add_foreign_key "quotes", "characters"
   add_foreign_key "reports", "users"
   add_foreign_key "reports", "users", column: "moderator_id"
-  add_foreign_key "review_likes", "reviews"
   add_foreign_key "review_likes", "users"
   add_foreign_key "reviews", "library_entries"
   add_foreign_key "streaming_links", "streamers"
```

Comment:
Can you rebase this out?  🤔 How did it even get in here?  You didn't do any migrations, why would you include `db/schema.rb`